### ISSUE TITLE
Current instance not highlighted if symlinked

### ISSUE THREAD

**Issue by pixelchutes** [meltingmedia/MODX-Shell#13]
If you have configured an instance using a symlink, `modx config:list` does not show the current instance highlighted when called within that symlinked directory.

Is it possible to support symlinked config paths?

NOTE: It doesn't seem to make a difference if called from the absolute path either.

**Comment by pixelchutes** [comment]
Hey @rtripault -- any pointers on where to look to extend this support to work with symlinks?

I think it may also have an issue allowing commands via a targeted site (e.g. `--site`)

`MODX Shell version 0.0.3-dev` 224ffd781a96997c30a4453881cf10e0d528bcab

**Comment by rtripault** [comment]
I guess you could try replacing the following method https://github.com/meltingmedia/MODX-Shell/blob/master/src/Configuration/Instance.php#L117

```php
public function findFormPath($path)
    {
        $path = realpath($path);
        foreach ($this->items as $name => $data) {
            if (array_key_exists('base_path', $data)) {
                $instancePath = realpath($data['base_path']);
                if (substr($path, 0, strlen($instancePath)) === $instancePath) {
                    return $name;
                }
            }
        }

        return null;
    }
```

Quick, dirty and not tested

**Comment by pixelchutes** [comment]
Thanks! I can confirm **it works** (highlights current instance when in same directory), however looks like I'll need to dig a little deeper to find out why site targeting (via instance name) is not working w/ symlinks:

```
> modx --site=somedomain.com version


  [InvalidArgumentException]
  Command "version" is not defined.
```

**EDIT:**

Maybe I was just calling it wrong. I was able to get this format to work! 😎 

`modx -s'somedomain.com' version`

**Comment by rtripault** [comment]
Thanks for confirming.

Closing this one as it should be fixed with a268c7091c4af636db207cdd260c79bba88a346e
